src/SpecFlow.NetCore/Fixer.cs

```diff
@@ -14,13 +14,15 @@ internal class Fixer
 		private readonly string _specFlowExe;
 		private string _testFramework;
 		private FileInfo[] _featureFiles;
+		private readonly string _toolsVersion;
 
-		public Fixer(string specFlowPath = null, string testFramework = null)
+		public Fixer(string specFlowPath = null, string testFramework = null, string toolsVersion = null)
 		{
 			_specFlowExe = FindSpecFlow(specFlowPath);
 			WriteLine("Found: " + _specFlowExe);
 
 			_testFramework = testFramework;
+			_toolsVersion = toolsVersion ?? "14.0";

```

Comment:
Can just set the `"14.0"` in the optional argument instead of having it as null?

Reply:
Of course. Adjusted accordingly.